bindings/azure/eventgrid/metadata.yaml

```diff
@@ -0,0 +1,68 @@
+# yaml-language-server: $schema=../../../component-metadata-schema.json
+schemaVersion: v1
+type: bindings
+name: azure.eventgrid
+version: v1
+status: beta
+title: "Azure Event Grid"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-bindings/eventgrid/
+binding:
+  input: true
+  output: true
+  operations:
+    - name: create
+      description: "Create an event subscription."
+capabilities: []
+builtinAuthenticationProfiles:
+  - name: "azuread"
+metadata:
+  # Required Input Binding Metadata
```

Comment:
For things that are only-input, please add:

```yaml
binding:
  input: true
  output: false
```

Same for output-only.

Otherwise, the default is to match the top-level `binding` property, which is input and output

Reply:
I hope I fixed this right on d675ab99f6b21933eb56ad1f39aaba92d9ab7601